composer.json

```diff
@@ -6,7 +6,7 @@
     "require": {
       "magento/framework": "*",
       "snowdog/frontools": "^1.6",
-      "snowdog/module-alpaca-components": "~1.4.0",
+      "snowdog/module-alpaca-components": "dev-develop",
```

Comment:
Why do you need this change?

Reply:
I got this change when I created `feature` branch from `Develop`.